dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/context/ContextManager.java

```diff
@@ -1056,6 +1063,15 @@ public InitializeResult initialize() throws WIMApplicationException {
             iEnvironment.put(LDAP_ENV_PROP_READ_TIMEOUT, String.valueOf(DEFAULT_READ_TIMEOUT));
         }
 
+        /*
+         * Enabled JNDI BER output if required.
+         */
+        if (iJndiOutputEnabled != null && iJndiOutputEnabled) {
+            synchronized (iLock) {
```

Comment:
Did I miss this synchronized block? I don't think we need this. We are adding a new instance of the output stream to the environment, so I don't think there are any race conditions here.